src/components/common/YourCompany/NameTranslations/CorrectNameTranslation.vue

```diff
@@ -1,42 +0,0 @@
-<template>
```

Comment:
This was an intermediate component, which is sometimes useful to bridge the high-level view (page) with a low-level component. One good use case is to have the store and actions here instead of the view or component -- especially if it's an external shared component. In this case, the component is local and already talked to the store, so THIS component is an unnecessary abstraction.